apps/docs-snippets/src/guide/contracts/call-parameters.test.ts

```diff
@@ -1,30 +1,34 @@
-import type { Contract, Provider } from 'fuels';
 import { BN } from 'fuels';
+import { launchTestNode } from 'fuels/test-utils';
 
-import { DocSnippetProjectsEnum } from '../../../test/fixtures/forc-projects';
-import { createAndDeployContractFromProject } from '../../utils';
+import { ReturnContextFactory } from '../../../test/typegen';
 
 /**
  * @group node
+ * @group browser
  */
-describe(__filename, () => {
-  let contract: Contract;
-  let provider: Provider;
-  let baseAssetId: string;
-  beforeAll(async () => {
-    contract = await createAndDeployContractFromProject(DocSnippetProjectsEnum.RETURN_CONTEXT);
-    provider = contract.provider;
-    baseAssetId = provider.getBaseAssetId();
-  });
-
+describe('Call Parameters', () => {
   it('should successfully execute contract call with forwarded amount', async () => {
+    using launched = await launchTestNode({
+      contractsConfigs: [
+        {
+          factory: ReturnContextFactory,
+        },
+      ],
+    });
+
+    const {
+      provider,
+      contracts: [contract],
+    } = launched;
```

Comment:
This pattern of deploying one contract pops up so much that I think we should create a test helper function that us and our users can use:

```ts
// packages/contract/src/test-utils/launch-test-contract.ts
import { launchTestNode, type DeployContractConfig } from './launch-test-node';

export async function launchTestContract<T extends DeployContractConfig>(config: T) {
  const {
    cleanup,
    contracts: [contract],
  } = await launchTestNode({ contractsConfigs: [config] });

  return Object.assign(contract, { [Symbol.dispose]: cleanup });
}
```

The usage would be:
```ts
using contract = await launchTestContract({
    factory: ReturnContextFactory,
});

const provider = contract.provider;
```

There are also plans to later simplify the `DeployContractConfig` so that you can pass the factory like this:
```ts
using contract = await launchTestContract(ReturnContextFactory)
```

Reply:
That's a good point @nedsalk perhaps we should put this in a follow-up issue as I suppose that we'd need to refactor quite a few of the tests with this new helper function and also document it for consumers.